src/main/java/com/google/cloud/tools/gradle/appengine/core/ShowConfigurationTask.java

```diff
@@ -106,12 +110,23 @@ private static String getFieldData(Field root, Object instance, int depth)
         .append(getGenericTypeData(root.getGenericType()))
         .append(") ")
         .append(root.getName())
-        .append(" = ")
-        .append(root.get(instance))
-        .append("\n");
+        .append(" = ");
+    try {
+      Method getter = instance.getClass().getMethod(getGetterFromFieldName(root.getName()));
+      result.append(getter.invoke(instance));
+    } catch (NoSuchMethodException ex) {
+      result.append(root.get(instance));
+    } catch (InvocationTargetException ex) {
```

Comment:
how does InvocationTargetException happen?

Reply:
If there's an exception thrown mid-invocation, it gets wrapped in an InvocationTargetException. In our case it can happen if someone runs showConfiguration w/ misconfigured project or version in appengine-web.xml.

Reply:
can you add a test to trigger this somehow?

Reply:
Sure